tests/basic/build/webpack.config.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
       description: 'My awesome Progressive Web App!',
       background_color: '#ffffff',
       ios: true,
+      crossorigin: 'use-credentials',
```

Comment:
perhaps you could remove the `crossorigin` key for the `basic` test `webpack.config.js` fixture? and reserve it for the `complex` test `webpack.config.js` fixture?